src/Client.php

```diff
@@ -26,6 +30,10 @@ class Client
     public Store $store;
     public EditableContent $editableContent;
     public Changes $changes;
+    public ReturnDto $returnDto;
```

Comment:
Myslím si, že $return použít můžeš, nejde použít jen Return jako class name, takže spíš 
```suggestion
    public ReturnDto $return;
```